packages/api/src/routes/metrics.js

```diff
@@ -1,99 +1,64 @@
-import { secrets, database } from '../constants.js'
-import { DBClient } from '../utils/db-client.js'
-
-const db = new DBClient(database.url, secrets.database)
-
-export async function getUserMetrics() {
-  const query = db.client.from('user')
-  const res = await query
-    .select('*', { head: true, count: 'exact' })
-    .range(0, 1)
-  if (res.error) {
-    throw res.error
-  }
-  return { total: res.count }
-}
-
-export async function getNftMetrics() {
-  const types = ['Car', 'Blob', 'Multipart', 'Remote', 'Nft']
-  const totals = await Promise.all(
-    types.map(async (t) => {
-      const query = db.client.from('upload')
-      const res = await query
-        .select('*', { head: true, count: 'exact' })
-        .filter('type', 'eq', t)
-        .range(0, 1)
-      if (res.error) {
-        throw res.error
-      }
-      return { [t]: res.count }
-    })
-  )
-  return { totals: Object.assign({}, ...totals) }
-}
-
-export async function getPinMetrics() {
-  const services = ['Pinata', 'IpfsCluster', 'IpfsCluster2']
-  const statuses = ['PinQueued', 'Pinning', 'Pinned', 'PinError']
-  const totals = await Promise.all(
-    services.map(async (service) => {
-      const totals = await Promise.all(
-        statuses.map(async (status) => {
-          const query = db.client.from('pin')
-          const res = await query
-            .select('*', { head: true, count: 'exact' })
-            .filter('service', 'eq', service)
-            .filter('status', 'eq', status)
-            .range(0, 1)
-          if (res.error) {
-            throw res.error
-          }
-          return { [status]: res.count }
-        })
-      )
-      return { [service]: Object.assign({}, ...totals) }
-    })
-  )
-  return { totals: Object.assign({}, ...totals) }
-}
+const UPLOAD_TYPES = ['Car', 'Blob', 'Multipart', 'Remote', 'Nft']
```

Comment:
nit: it would be great to have these in a single place. Currently, we have them at list here and in the cron code. This might be error prone in the case of a new PinService, or type of upload as we can forget to update the other side.

Reply:
We already do this in the cron package:

```js
import { DBClient } from '../../../api/src/utils/db-client.js'
```

So I think should be ok to import them...